Shopery/apps/fe-admin/src/app/nx-welcome.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '@shopery/libs';
```

Comment:
```suggestion
import { Button } from '@shopery/ui';
```
You can make it with tsconfig